autowebcompat/utils.py

```diff
@@ -293,3 +293,34 @@ def write_train_info(information, model, train_history, file_name=None):
             row = train_history_list[i]
             for col in row:
                 print('%f\t\t' % col, end=' ', file=f)
+
+
+def create_screenshot_name(file_name_info):
```

Comment:
You can pass several arguments here instead of an object, it's cleaner

Reply:
seq_no can have the default of None

Reply:
Should I just pass them differently? Can I store them as dictionary in collect.py?

Reply:
And let's call the function `create_file_name`, to match the equivalent but opposite `parse_file_name`.

Reply:
> Should I just pass them differently? Can I store them as dictionary in collect.py?

What do you mean?

Reply:
I am creating an object like this
> image_file_info['folder'] = 'data'
image_file_info['bug_id'] = str(bug['id'])
 image_file_info['browser'] = browser

in collect.py 
Should I keep it this way only? And when calling the above function just call them like
> create_screenshot_name(image_file_info['folder'],  image_file_info['bug_id'],  image_file_info['width'],  image_file_info['height'])

Or should I keep different variables for them?

Reply:
Keep different variables, the argument list will get longer (but not too long) but you won't needlessly create an object